src/components/Header.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+const StyledHeader = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: #4281a4;
+  color: #fff;
+`;
+
+const Headline = styled.h1`
+  font-size: 20px;
+`;
+
+const StyledLogo = styled.img`
+  margin: 10px;
+`;
+
+function Header(props) {
+  return (
+    <StyledHeader>
+      <StyledLogo src="icon_file.png" />
+      <Headline>{props.title}</Headline>
```

Comment:
du kannst hier auch das props weglassen und nur {title} schreiben. dann müsstest du in Zeile 21 auch {title} einfügen statt props. Ist nur ne andere Schreibweise.